run.js

```diff
@@ -73,6 +74,20 @@ async function main () {
 async function parseExperimentsConfig (config) {
   const experiments = []
 
+  for (const expConfig of config.demo) {
+    const params = {
+      nrBlocks: expConfig.nrBlocks,
+      blockByteSize: expConfig.blockByteSize
+    }
+
+    experiments.push({
+      experimentClass: DriveGetExperiment,
+      params,
+      name: 'demo',
```

Comment:
rename, and for the description 'Read hyperdrive files'

Reply:
Corrected